### ISSUE TITLE
Publish new release

### ISSUE THREAD

**Issue by andrcuns** [piotrmurach/tty-spinner#47]
Hi,

I have been using spinner in one of my projects and it works great, however I could really use some of the latest `master` branch additions, mainly:

* `log` method to print above spinner
* `tty?` method which seems to be private in the released version.

Have you considered pushing a new release any time soon?

**Comment by piotrmurach** [comment]
Hi Andrejs 👋 

Thanks for using this gem in your project.

My plan was to look at the [timer](https://github.com/piotrmurach/tty-spinner/issues/44) and [multi-spinner](https://github.com/piotrmurach/tty-spinner/issues/45) issues. However, there is no reason why I cannot make this release earlier. I will see what I can do.

**Comment by piotrmurach** [comment]
@andrcuns This is still on my list and I plan to make a release this coming weekend.

**Comment by jgarth** [comment]
Hi @piotrmurach! Thanks for making this (and the other cool parts of `tty`). I'd also benefit from a new release, instead of relying on `master` in my Gemfiles. Can I help with the release somehow?

**Comment by piotrmurach** [comment]
@jgarth Yes, you can. The `log` method isn't fully robust for release. Here's the [Spinner breaks on multiline message](https://github.com/piotrmurach/tty-spinner/issues/43) issue that needs resolving. My last comment summarises the problem and possible solutions. Once this is addressed, I think we will be in a good place to cut a new release.

**Comment by MadBomber** [comment]
If the methods are not available, the README documentation should at least reflect that they are either not available but planned or they should be. removed from the README completely.

I added this patch to my code for a log method:

```ruby
module TTY
  class Spinner
    # Log a message to the output
    # This will clear the current spinner line, print the log message,
    # and then redraw or resume the spinner on a new line.
    #
    # @param [String] message
    #   the log message to print
    #
    # @api public
    def log(message)
      synchronize do
        clear_line    # Clear the spinner
        output.puts(message) # Log the message
        redraw_indent # Redraw the spinner frame
      end
    end
  end
end
```

**Comment by capripot** [comment]
Hi @piotrmurach Thank you for your messages! What could move this issue forward?

**Comment by dmorgan-fa** [comment]
@piotrmurach, thanks for all of the work on the fleet of `tty` tools, they're fantastic!
Is there any chance of getting a release for this gem to support `.log`?
